### ISSUE TITLE
how to fix the position of "Show the menu directly when selecting text." Pop-up.

### ISSUE THREAD

**Issue by geaty3** [Pitmairen/selection-search#139]
Hey, thanks for the great extension.
Is there a way to fix the position of Pop-up when the option "Show the menu directly when selecting text." is enabled?
It will help prevent unintended pop-up that follows my cursor when clicking link or java.

**Comment by Pitmairen** [comment]
I don't think it is possible at the moment, but it should be fairly simple to fix this.

If the show directly option is not checked, there is this option to show the popup button relative to the mouse. We can make this option also apply when the show directly options is used.

![image](https://github.com/Pitmairen/selection-search/assets/357032/8610ebfc-7adc-4d73-86d7-3baa8b4c2c7c)

I will try to get this into the next version.

**Comment by geaty3** [comment]
Great to hear that.
It will be even more perfect if the pop-up appears just below the selection so the pop-up won't block the selected text, like adding "transform: translate(%, px)" in the CSS.
